lib/perfBaseTest.py

```diff
@@ -53,7 +53,7 @@ def setUp(self):
         self.target_helper = targetHelper.TagetHelper(self.env)
 
         # init sikuli
-        self.sikuli = sikuli.Sikuli()
+        self.sikuli = sikuli.Sikuli(self.env.run_sikulix_cmd_path, self.env.hasal_dir)
```

Comment:
I didn't see you commit Environment.py in. There is no such default variable currently.